public/modules/core/views/home.client.view.html

```diff
@@ -90,14 +93,74 @@
 			<h2>MATCHING TRIALS | {{filteredTrials.length}}</h2><br/>
 			<a data-ng-repeat="trial in filteredTrials = (trials | filter: searchCriteria())  " data-ng-href="#!/trials/{{trial.nctId}}" class="list-group-item">
 				<div>
-					<h3 data-ng-bind="trial.title"></h3>
+					<h3>{{trial.title}}</h3>
 					<h3><small><p class="lead">{{trial.nctId}} <br/> {{trial.phase}}<br/> {{trial.recruitingStatus}}<br/> {{trial.countries.join(', ')}}</p></small></h3>
+
+					<div ng-if="trial.fetchedAlterations.length > 0" ng-repeat="x in trial.fetchedAlterations" >
+						<h1><small>
+							<span style="float:left; margin-bottom: 10px" class="label label-warning"  ng-if="x.status === 'unconfirmed'" >{{x.gene+ "     " + x.alteration}}</span>
+							<span style="float:left;" class="label label-success" ng-if="x.curationMethod === 'manually' || x.status === 'confirmed'">{{x.gene + "     " + x.alteration}}</span>
+
+						</small></h1>
+
+					</div>
+					<br/><br/><br/>
+					<div ng-if="trial.fetchedAlterations.length === 0" >
+						<p>No Alteration Info Available at this time</p>
+					</div>
+
+
 				</div>
 			</a>
 
 		</div>
 
 	</div>
+<br/><br/><br/>
 
-</section>
 
+	<div ng-show="!showRefine">
+	<div class="panel panel-default" style="width: 45%; float:left; margin:10px; ">
+		<div class="panel-heading">Total # of Genomically Defined Clinical Trails: {{trialsMappingCount}}</div>
+		<div class="panel-body">
+			<!--<select chosen multiple ng-options="x as x for x in ['Not yet recruiting', 'Recruiting', 'Enrolling by invitation', 'Active, not recruiting', 'Completed', 'Others' ]" class="multiSelect" data-placeholder="Select status" ng-model="sort" ng-change="getCriteria1(country, country, 'country')">-->
+			<!--</select>-->
+
+			<!--<div class="radio" style="float:right">-->
+				<!--<label><input type="radio" ng-model="sort1" value="4" ng-change="getCriteria1('all', status, 'status')">Descending</label><br/>-->
+				<!--<label><input type="radio" ng-model="sort1" value="1" ng-change="getCriteria1(false, status, 'status')">Alphabetical</label><br/>-->
+			<!--</div>-->
+
+
+			<div style="overflow-x: auto" id="geneTrails">
+				<img src="modules/core/img/loaders/loading.gif" style="width: 100px; height: 100px;" ng-if="loadingGeneData">
+			</div>
+		</div>
+
+		<br/>
+	</div>
+
+	<div class="panel panel-default" style="width: 45%; float:left;margin:10px">
```

Comment:
This style has been repeated several times, you could put them into css file and give class name instead.